dir.props

```diff
@@ -332,6 +332,7 @@
   <!-- Change define constants as needed -->
   <PropertyGroup Condition="'$(MsbuildDebugger)' == 'true' AND '$(MonoBuild)' != 'true'">
     <DefineConstants>$(DefineConstants);FEATURE_MSBUILD_DEBUGGER</DefineConstants>
+    <DefineConstants Condition="$(LocalizedBuild) == 'true'">$(DefineConstants);LOCALIZED_BUILD</DefineConstants>
```

Comment:
Yikes, yeah--that's obviously wrong.

It's talking about the half-implemented feature to step through MSBuild builds in the VS debugger, which is off in all of our builds.